external/storm-kafka/src/jvm/org/apache/storm/kafka/PartitionManager.java

```diff
@@ -46,6 +46,9 @@
     private final CountMetric _fetchAPIMessageCount;
```

Comment:
@harshach - is it fine with you to do the refactoring separately? Apologies for a long pause.